pyodide_build/tests/test_cli.py

```diff
@@ -23,6 +23,16 @@
 RECIPE_DIR = Path(__file__).parent / "recipe" / "_test_recipes"
 
 
+def assert_runner_succeeded(result):
+    __tracebackhide__ = True
+    print(result.stdout)
```

Comment:
```suggestion
```

Reply:
I think this is helpful? In my opinion, either the test is passing and stdout gets hidden or its failing and stdout is useful.

Reply:
I, too, think that this would be helpful – though it would be hidden anyway unless we use the `-s` option with `pytest`; [we currently use `-vra`](https://github.com/pyodide/pyodide-build/blob/147c1557adaccb5ab2356b7ed4c334edb04c7596/pyproject.toml#L76).

Reply:
Yeah it's preferable for it to be hidden by default.